learning-bitmark/property-system-introduction/problem-we-are-trying-to-solve.md

```diff
@@ -0,0 +1,47 @@
+# Bitmark Property System Introduction 
+
+Digital assets and data feel ephemeral. It's easy to lose track of your digital assets; and it's easy to let social networks and other content aggregators claim ownership over your data. Hackers and 'bots can steal your digital properties, and today you might not have much recourse.

```

Comment:
1) The phrase "Digital assets and data feel ephemeral." alone itself does not makes too much sense for me, maybe add a comma and continue with "it's easy to....." looks a bit better.
2) Also I don't know why it is 'bots and not only bots.
3) "...you might not have much recourse" for? Retrieve the data? Claim ownership? There is something missing after "recourse".